selfdrive/car/hyundai/values.py

```diff
@@ -1940,6 +1942,23 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x00MQhe SCC FHCUP      1.00 1.07 99110-P4000         ',
     ],
   },
+  CAR.AZERA_2022: {
+    (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x00IG__ SCC F-CU-      1.00 1.00 99110-G8100         ',
+    ],
+    (Ecu.eps, 0x7d4, None): [
+      b'\xf1\x00IG  MDPS C 1.00 1.02 56310G8510\x00 4IGSC103',
+    ],
+    (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00IG  MFC  AT MES LHD 1.00 1.04 99211-G8100 200511',
+    ],
+    (Ecu.transmission, 0x7e1, None): [
+      b'\xf1\x00bcsh8p54  U912\x00\x00\x00\x00\x00\x00SIG0M35MH0\xa4 |.',
+    ],
+    (Ecu.engine, 0x7e0, None): [
+      b'\xf1\x81641KA051\x00\x00\x00\x00\x00\x00\x00\x00',
+    ],
+  },
```

Comment:
No ABS?

Reply:
Yep, no ABS FW.

Reply:
Know why?

Reply:
Not really sure, did a quick search and this car does have ABS (whew).

Reply:
It's definitely there, it just requires a different query `iso-tp query bad response: (2001, None) - 0x7f2231`.